Tasks/DeployVisualStudioTestAgent/TestAgentHelper.ps1

```diff
@@ -204,5 +204,24 @@ namespace CapabilityHelpers.VisualStudio.Setup.Com
     }
 }
 
-
-
+function Remove-Service([String] $ServiceName)
+{
+    if(Get-Service $ServiceName -ErrorAction SilentlyContinue)
+    {
+        $service = (Get-WmiObject Win32_Service -filter "name='$ServiceName'")
+        Write-Verbose -Message("Trying to delete service {0}" -f $ServiceName) -Verbose
+        if($service)
+        {
+            $service.StopService()
+            $deleteServiceCode = $service.Delete()
+            if($deleteServiceCode -eq 0)
```

Comment:
>f($deleteServiceCode -eq 0) [](start = 13, length = 27)

As per [https://msdn.microsoft.com/en-us/library/aa389960(VS.85).aspx](https://msdn.microsoft.com/en-us/library/aa389960(VS.85).aspx) Delete method returns 0 if the request was accepted.